bin/slackin

```diff
@@ -6,7 +6,7 @@ var slackin = require('./../node').default;
 
 args
   .option(['p', 'port'], 'Port to listen on [$PORT or 3000]', require('hostenv').PORT || process.env.PORT || 3000)
-  .option(['h', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
+  .option(['H', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
```

Comment:
I agree that there should be different flags used for help and hostname, just wanna point out that help wasn't getting open by default previously when passing `-h`.

`-h` would always be interpreted as hostname, see this comparison of the commit just before the refactor and the commit just after:

**Old version (4ea80c42)**
```
$ bin/slackin -h
error: option `-h, --hostname <hostname>' argument missing

$ bin/slackin -h "0.0.0.0" -p 8888 [subdomain] [api_key] 
Thu Jun 23 2016 19:41:15 GMT+1000 (AEST) – fetching
Thu Jun 23 2016 19:41:15 GMT+1000 (AEST) – listening on 0.0.0.0:8888
Thu Jun 23 2016 19:41:16 GMT+1000 (AEST) – ready
```

**New version (15126845b42b50632962bb887a2876a25bf136db)**
```
$ bin/slackin -h
bin/slackin -h

  Usage: slackin [options] [command] <team-id> <api-token>

  Commands:

    help  Display help

  Options:

    -c, --channels          One or more comma-separated channel names to allow single-channel guests [$SLACK_CHANNELS]
    -C, --coc               Full URL to a CoC that needs to be agreed to
    -S, --css               Full URL to a custom CSS file to use on the main page
    -h, --help              Output usage information
    -h, --hostname [value]  Hostname to listen on [$HOSTNAME or 0.0.0.0]
    -i, --interval <n>      How frequently (ms) to poll Slack [$SLACK_INTERVAL or 5000]
    -P, --path [value]      Path to serve slackin under
    -p, --port <n>          Port to listen on [$PORT or 3000]
    -s, --silent            Do not print out warns or errors
    -v, --version           Output the version number



$ bin/slackin -h "0.0.0.0" -p 8888 [subdomain] [api_key]

  Usage: slackin [options] [command] <team-id> <api-token>

  Commands:

    help  Display help

  Options:

    -c, --channels          One or more comma-separated channel names to allow single-channel guests [$SLACK_CHANNELS]
    -C, --coc               Full URL to a CoC that needs to be agreed to
    -S, --css               Full URL to a custom CSS file to use on the main page
    -h, --help              Output usage information
    -h, --hostname [value]  Hostname to listen on [$HOSTNAME or 0.0.0.0]
    -i, --interval <n>      How frequently (ms) to poll Slack [$SLACK_INTERVAL or 5000]
    -P, --path [value]      Path to serve slackin under
    -p, --port <n>          Port to listen on [$PORT or 3000]
    -s, --silent            Do not print out warns or errors
    -v, --version           Output the version number
```

I'm worried that changing `-h` to `-H` for hostname will break previously working deployments in favour of clearer CLI options - though that might be a tradeoff @rauchg is happy with.